src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -1037,15 +1038,25 @@ cu_stat(struct sched_cmd *cmd)
 	pkg->data[pkt_idx++] = exec->num_slots;
 
 	/* number of CUs */
-	pkg->data[pkt_idx++] = exec->num_cus;
+	pkg->data[pkt_idx++] = exec->num_cus + sk->sk_ncus;
 
 	/* individual CU execution stat */
 	for (i = 0; i < exec->num_cus && pkt_idx < max_idx; ++i)
 		pkg->data[pkt_idx++] = exec->zcu[i].usage;
 
+	for (i = 0; i < sk->sk_ncus && pkt_idx < max_idx; ++i)
+		pkg->data[pkt_idx++] = sk->sk_cu[i]->usage;
```

Comment:
We probably need to 
1. check if sk is valid (rare case but could happen if system is lack of memory)
2. hold sk mutex (sk->sk_lock) to prevent sk or individual scu is released (due to scu crash)

Reply:
done